src/store/modules/auth.store.js

```diff
@@ -50,8 +50,52 @@ export const authStore = {
                         dispatch('alertStore/error', error, { root: true });
                     }
                 );
-        }     
+        },
+
+        checkUser( {dispatch, commit}, userId) {
+            authService.checkUser(userId)
+            .then(
+                () => {
+                    
+                },
+                error => {
+                    commit('loginFailure');
+                    dispatch('alertStore/error', error, { root: true });
+                }
+            ); 
+        },
+            
+        OTP() {
```

Comment:
please use verbose name for functions like `sendOTP` which denotes the purpose. Just `OTP` denotes a variable name that holds OTP. 

Please follow such ideologies in future for any kind of project whether it be college or professional.